docs/data/data-grid/column-definition/AutogeneratedRows.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react';
+import {
+  DataGridPremium,
+  GridColDef,
+  isAutogeneratedRow,
+  useGridApiRef,
+  useKeepGroupedColumnsHidden,
+} from '@mui/x-data-grid-premium';
+import { useMovieData } from '@mui/x-data-grid-generator';
+
+export default function AutogeneratedRows() {
+  const data = useMovieData();
```

Comment:
```suggestion
  const { rows } = useMovieData();
```

To make it more explicit that it's only used for generating rows.